apps/api-catalogue/components/FocusableClickBox/FocusableClickBox.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {
+  ElementType,
+  AllHTMLAttributes,
+  forwardRef,
+  useContext,
+} from 'react'
+import cn from 'classnames'
+import { useToggle } from 'react-use'
+import {
+  Link,
+  Box,
+  UseBoxStylesProps,
+  ColorSchemeContext,
+  ColorSchemes,
+} from '@island.is/island-ui/core'
+
+import * as styles from './FocusableClickBox.treat'
+
+interface Props
+  extends Omit<UseBoxStylesProps, 'component'>,
+    Omit<AllHTMLAttributes<HTMLElement>, 'width' | 'height' | 'className'> {
+  component?: ElementType
+  color?: ColorSchemes
+  //onClick?: React.MouseEvent<HTMLElement, MouseEvent>;
+  onClick?: () => void
+}
+
+// FocusableClickBox is a wrapper component that handles focus styles.
+// Most props are forwarded to Box.
+// By default it renders as a Link component.
+
+export const FocusableClickBox = forwardRef<HTMLElement, Props>(
```

Comment:
This is mostly island-ui's FocusableBox. Should've added a click functionality to that one instead of copying it.